templates/include/remote/library.html

```diff
@@ -63,7 +63,7 @@
 	{% endif %}
 </li>
 {% endfor %}
-{% if more_music != None %}
+{% if more_musics|yesno:"yeah,no" == "no" %}
```

Comment:
It's the correct way to evaluate a True/False/None variable in django templating.
See : https://docs.djangoproject.com/en/1.7/ref/templates/builtins/#yesno